src/script/conversation/MessageRepository.ts

```diff
@@ -1168,16 +1167,9 @@ export class MessageRepository {
     return recipients;
   }
 
-  /**
-   * Sends a generic message to a conversation.
-   *
-   * @param eventInfoEntity Info about event
-   * @param skipLegalHold Skip the legal hold detection
-   * @returns Resolves when the message was sent
-   */
-  private async sendGenericMessage(eventInfoEntity: EventInfoEntity, skipLegalHold = false): Promise<ClientMismatch> {
+  private async sendGenericMessage(eventInfoEntity: EventInfoEntity, checkLegalHold: boolean): Promise<ClientMismatch> {
     try {
-      await this.grantOutgoingMessage(eventInfoEntity, undefined, skipLegalHold);
+      await this.grantOutgoingMessage(eventInfoEntity, undefined, checkLegalHold);
```

Comment:
`isLegalHold` maybe?

Reply:
No. It's a boolean that triggers if legal hold checks should be made at all.